modules/custom/d_social_media/d_social_media.module

```diff
@@ -14,3 +14,13 @@ function d_social_media_theme($existing, $type, $theme, $path) {
     ],
    ];
 }
+
+/**
+ * Implements hook_block_structure_alter().
+ */
+function d_social_media_block_structure_alter(&$structure, $context) {
```

Comment:
Te bloki powinny się znaleźć w d_demo, chodzi o to że bez włączania d_demo nie powinno być treści demonstracyjnych

Reply:
Skąd pochodzi ten hook? W core ani w tym projekcie nie znalazłem jego implementacji

Reply:
@aliegmann-droptica here's the custom hook added https://github.com/droptica/droopler/blob/master/modules/custom/d_content/modules/d_content_init/d_content_init.module#L334